RPLCD/lcd.py

```diff
@@ -27,7 +27,12 @@
 from . import codecs
 from . import common as c
 from .compat import range
-
+import sys
+if sys.version_info.major < 3:
+    from time import clock as now
+else:
+    from time import perf_counter as now
```

Comment:
Looking at this Python 2 compat code, I think it's time to drop Python 2 support in the next release. However, for now, let's keep it since it still works.